### PR TITLE
bump mmda inside symbolscraper-service

### PR DESCRIPTION
**Pull request by yoganandc**
i also made it log less because i want these logs going to datadog. motivating reason was a bug fix that was made to symbolscraper after i built the last image a while ago.

https://github.com/allenai/scholar/issues/28668

### CODE REVIEW

File: mmda/parsers/symbol_scraper_parser.py [hunk 1]
```diff
@@ -245,6 +248,10 @@ def _convert_nested_text_to_doc_json(self, page_to_row_to_tokens: Dict) -> Dict:
             Rows: [row.to_json() for row in row_annos]
         }
 
+    def print(self, msg):
```
Comment (rodneykinney) [inline mmda/parsers/symbol_scraper_parser.py:251]: I would prefer to handle this using the standard Python `logging` module.

File: mmda/parsers/symbol_scraper_parser.py [hunk 2]
```diff
@@ -262,13 +272,13 @@ def _parse_xml_to_doc(self, xmlfile: str) -> Document:
...
         else:
-            self.print(f'Symbol Scraper took {runtime} sec for {xmlfile}...')
+            self._log(f'Symbol Scraper took {runtime} sec for {xmlfile}...')
 
         # get page metrics
         page_to_metrics = self._parse_page_to_metrics(xml_lines=xml_lines)
-        self.print(f'\tNum pages: {len(page_to_metrics)}')
-        self.print(f"\tAvg tokens: {sum([metric['tokens'] for metric in page_to_metrics.values()]) / len(page_to_metrics)}")
-        self.print(f"\tAvg rows: {sum([metric['rows'] for metric in page_to_metrics.values()]) / len(page_to_metrics)}")
+        self._log(f'\tNum pages: {len(page_to_metrics)}')
```
Comment (rodneykinney) [inline mmda/parsers/symbol_scraper_parser.py:279]: Just use `logger.debug` here.

File: services/parsers/symbolscraper/model/predictor.py [hunk 3]
```diff
@@ -41,7 +41,7 @@ def __init__(self, config: PredictorConfig):
         Initialize your model using the passed parameters
         """
         self._config = config
-        self._sscraper = SymbolScraperParser("sscraper", print_info=False)
+        self._sscraper = SymbolScraperParser("sscraper", debug=False)
```
Comment (rodneykinney) [inline services/parsers/symbolscraper/model/predictor.py:44]: Here you can call `symbol_scraper_parser.logger.setLevel(logging.DEBUG)` based on config.

Comment (yoganandc) [reply services/parsers/symbolscraper/model/predictor.py:44]: ok

File: mmda/parsers/symbol_scraper_parser.py [hunk 4]
```diff
@@ -255,13 +264,13 @@ def _parse_xml_to_doc(self, xmlfile: str) -> Document:
...
+            logger.info(f'Symbol Scraper took {runtime} sec for {xmlfile}...')
 
         # get page metrics
         page_to_metrics = self._parse_page_to_metrics(xml_lines=xml_lines)
-        print(f'\tNum pages: {len(page_to_metrics)}')
-        print(f"\tAvg tokens: {sum([metric['tokens'] for metric in page_to_metrics.values()]) / len(page_to_metrics)}")
-        print(f"\tAvg rows: {sum([metric['rows'] for metric in page_to_metrics.values()]) / len(page_to_metrics)}")
+        logger.info(f'\tNum pages: {len(page_to_metrics)}')
+        logger.info(f"\tAvg tokens: {sum([metric['tokens'] for metric in page_to_metrics.values()]) / len(page_to_metrics)}")
+        logger.info(f"\tAvg rows: {sum([metric['rows'] for metric in page_to_metrics.values()]) / len(page_to_metrics)}")
```
Comment (yoganandc) [inline mmda/parsers/symbol_scraper_parser.py:273]: i want the default case to be that messages are printed

**Review by rodneykinney — APPROVED** [review]
👍

**Pull request merged**